samples/Mvc.DynamicCorsPolicyProvider/Mvc.DynamicCorsPolicyProvider.sln

```diff
@@ -1,22 +0,0 @@
-﻿
```

Comment:
Intentional?

Reply:
Yeah, pretty sure we don't have xproj anymore or require a sln file with this one project.